x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/data/DoubleBlockBuilder.java

```diff
@@ -18,40 +22,80 @@
  */
 final class DoubleBlockBuilder extends AbstractBlockBuilder implements DoubleBlock.Builder {
 
-    private double[] values;
+    private interface DoubleValues extends Releasable {
+        void setValue(int index, double value);
+    }
```

Comment:
I worry about these things slowing things down by making the write bimorphic instead of an array access.

Reply:
I've pushed https://github.com/elastic/elasticsearch/pull/103340/commits/5e5330dda707e843525af5745267d91bd46e8e7f.